workflows/test.yml

```diff
@@ -0,0 +1,25 @@
+name: Tests
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.5, 3.10]
```

Comment:
add 3.9 for a good measure?

Reply:
done. By the way, I don't have permissions to merge this PR